features/waarden.feature

```diff
@@ -96,20 +143,39 @@ Feature: leveren van waarden bij een WOZ-object
     ]
     """
 
-  Scenario: Belanghebbende krijgt verlaging van de WOZ-waarde na bezwaar: bij beschikkingen met zelfde waardepeildatum en zelfde ingangsdatum gebruiken we de meest recente beschikking
-    Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde op 01-01-2017
-    En dat bezwaar is afgehandeld met de status "bezwaar afgehandeld, vastgestelde waarde veranderd"
-    En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde op 01-01-2018
-    En dat bezwaar heeft de status "bezwaar ingediend"
-    En WOZ-object met objectnummer "082600014669" heeft de volgende beschikkingen:
-      | Peildatum  | WOZ-waarde   | Belanghebbende | Ingangsdatum | Datum beschikking |
-      | 01-01-2019 | 195.000 euro | R. Geels       | 01-01-2020   | 13-01-2020        |
-      | 01-01-2017 | 152.000 euro | R. Geels       | 01-01-2018   | 03-02-2019        |
-      | 01-01-2018 | 176.000 euro | R. Geels       | 01-01-2019   | 14-01-2019        |
-      | 01-01-2017 | 171.000 euro | R. Geels       | 01-01-2018   | 15-01-2018        |
-      | 01-01-2016 | 164.000 euro | R. Geels       | 01-01-2017   | 16-01-2017        |
-    Als het WOZ object wordt opgevraagd met /wozobjecten/082600014669
-    Dan zijn de waarden:
+  Scenario: Belanghebbende krijgt verlaging van de WOZ-waarde na bezwaar
+    Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde met peildatum 01-01-2017
```

Comment:
Is het relevant om in de gegeven op te nemen? Het bepalen van de relevante waarden wordt toch alleen afgeleid uit de geleverde waarden

Reply:
het doel van de feature is niet alleen de werking duidelijk maken voor de developer, maar ook om de werking duidelijk te maken voor meer functioneel ingestelde personen, zodat zij kunnen toetsen dat de beschreven functionaliteit ook gewenst is. Daarom heb ik hier eerst de onderliggende gebeurtenissen in de werkelijkheid toegevoegd, en in de laatste gegevensegel de technische consequenties die voor de developer van de SDK relevant zijn.

Is dat verwarrend of naar jouw mening niet nodig?

Reply:
Ik vind het verwarrend omdat met de Gegeven stappen de context wordt beschreven. Nu lijkt het op: én de belanghebbende bezwaar heeft gemaakt én het WOZ object heeft de aangegeven waarden, dan is de relevante waarden gelijk aan de waarden opgegeven in de Dan stap. Misschien is het dan beter om de response die je krijgt ook in een Dan stap op te nemen. Ik denk aan zoiets:
Gegeven WOZ-object met objectnummer "082600014669" heeft de belanghebbende eigenaar bezwaar ingediend over de WOZ-waarde met peildatum 01-01-2017
En dat bezwaar is afgehandeld met de status "12: bezwaar afgehandeld, vastgestelde waarde veranderd"
En de belanghebbende eigenaar heeft bezwaar ingediend over de WOZ-waarde met peildatum 01-01-2018
En dat bezwaar heeft de status "10: bezwaaringediend"
Als het WOZ object wordt opgevraagd met /wozobjecten/082600014669
Dan zijn de geleverde waarden
{
...
}
en zijn de berekende/bepaalde relevante waarden
{
...
}

Reply:
@MelvLee goed advies, ik heb deze overgenomen, zie meest recente versie.